README.md

```diff
@@ -4,7 +4,11 @@
 
 Use `dartdoc` to generate HTML documentaton for your Dart package.
 
-For informtion about contributing to the dartdoc project, see the [contributor docs][].
+For information about contributing to the dartdoc project, see the [contributor docs][].
+
+To see issues and code for dartdoc servers, see
```

Comment:
"dartdoc servers" might be misinterpreted (does dartdoc ship an HTTP server?)

Maybe "hosted Dart API documentation" ?